test/bigint.cc

```diff
@@ -1,11 +1,12 @@
+// currently experimental guard with version of NODE_MAJOR_VERSION that it is
```

Comment:
```suggestion
// currently experimental guard with version of NAPI_VERSION that it is
```

Reply:
I think experimental feature tests do tell which Node.js versions it could be run on. And we should continuously update the test table on each backport.

And experimental features were always temporary status, IMO. It is not necessary to point out that an experimental guard flag is temporary, IMO.

Reply:
I think it is important to point out that we should NOT use NODE_MAJOR_VERSION once it is not experimental.

I'd be ok with:

// currently experimental temporarily guard with NODE_MAJOR_VERISION.
// Once its no longer experimental guard with version of NAPI_VERSION that it is release in.